scanner/src/main/kotlin/Scanner.kt

```diff
@@ -169,7 +176,8 @@ class Scanner(
             config = filteredScannerConfig,
             provenances = projectResults.provenances + packageResults.provenances,
             scanResults = projectResults.scanResults + packageResults.scanResults,
-            files = projectResults.files + packageResults.files
+            files = projectResults.files + packageResults.files,
+            scanners = scannersByPackages
```

Comment:
I believe line 180 should be `projectResult.scanners + packageResults.scanners`.

(Which implies the logic you added above should instead go into the other overload of `scan()`)

What do you think?

Reply:
Yes it is better.